tasks/base_tasks.py

```diff
@@ -430,6 +438,33 @@ def output(self):
         return LocalTarget(os.path.join('tmp', classpath(self), self.task_id))
 
 
+class DownloadUnzipTask(DownloadUncompressTask):
+    '''
+    Download a zip file to location {output}.zip and unzip it to the folder
+    {output}.  Subclasses only need to define a
+    :meth:`~.tasks.DownloadUnzipTask.download` method.
+    '''
+
+    def uncompress(self):
+        shell('unzip -d {output} {output}.zip'.format(output=self.output().path))
```

Comment:
Oh maaaaaan, remove that shell pls xD use https://docs.python.org/3.5/library/zipfile.html

Reply:
Done, using `zipfile` to unzip now :)